src/main/java/com/bruberu/gregtechfoodoption/GTFOMaterialHandler.java

```diff
@@ -171,6 +174,11 @@
     public static final GTFOOredictItem.OreDictItem CutCurd = new GTFOOredictItem.OreDictItem(1107, "cut_curd", 0xede3cc, MaterialIconSet.SHINY, GAEnums.GAOrePrefix.round);
     public static final GTFOOredictItem.OreDictItem CookedCurd = new GTFOOredictItem.OreDictItem(1108, "cooked_curd", 0xffe8b3, MaterialIconSet.SHINY, GAEnums.GAOrePrefix.round);
     public static final GTFOOredictItem.OreDictItem SaltedCurd = new GTFOOredictItem.OreDictItem(1109, "salted_curd", 0xf7d68b, MaterialIconSet.SHINY, GAEnums.GAOrePrefix.round);
+    public static final GTFOOredictItem.OreDictItem GorgonzolaCurd = new GTFOOredictItem.OreDictItem(1110,"gorgonzola_curd",0xe5e5f5, MaterialIconSet.SHINY, GAEnums.GAOrePrefix.fuelPure);
```

Comment:
For your OreDictItems and FluidMaterials, make sure to add spaces between all elements so that it's more readable.